proxy/pkg/config/default.go

```diff
@@ -0,0 +1,57 @@
+package config
+
+// Default are values stored in the flagset, but moved to a struct.
+func DefaultConfig() Config {
+	return Config{
```

Comment:
This is not compatible with flaex. We need to change the config parser and then make this change to every extension.

Reply:
![Screenshot 2021-08-01 at 23-25-32 Configuration ownCloud](https://user-images.githubusercontent.com/4378513/127785760-3f44e48b-02bb-4de6-b8d8-786c177ecd35.png)